jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -48,6 +51,9 @@ def buildExclusions = [
   [ /^cross-compiler-armv[67]-gcc-4.9/, anyType,   lt(10)  ],
   [ /^cross-compiler-armv[67]-gcc-4.9/, anyType,   gte(12) ],
   [ /^cross-compiler-armv[67]-gcc-6/, anyType,     lt(12)  ],
+  [ /^cross-compiler-armv[67]-gcc-6/, anyType,     gte(14) ],
+  [ /^cross-compiler-armv[67]-gcc-8/, anyType,     lt(14)  ],
```

Comment:
oh this is interesting, do we have that running? I see in https://github.com/nodejs/build/blob/master/ansible/roles/cross-compiler/files/cc-selector.sh I set it up to possibly work and I did do a toolchain for gcc-8 but don't recall ever getting it properly running.

This line should drop the `[67]` and just go with `7`. I think we probably should have done that with the gcc-6 one too since we dropped armv6 for Node 12 but there may be unintended consequences with doing that.

Reply:
I've added the `cross-compiler-armv7-gcc-8` to the cross compilers in [ci-release](https://ci-release.nodejs.org/label/cross-compiler-armv7-gcc-8/) and [ci](https://ci.nodejs.org/label/cross-compiler-armv7-gcc-8/) but they'll need to be added to iojs+release and whatever we do for node-test-binary-arm. I'll be interested to see if they actually work and I'll also need to revisit the configs I set up to make the crosstools! Probably even needs a re-do with a newer gcc-8: https://github.com/rvagg/rpi-newer-crosstools/blob/master/x64-gcc-8.3.0.config

Or maybe we should pull this bit out of this PR and handle it separately?

Reply:
I'll pull it

Reply:
PTAL